example_configs/keycloak_oidc/oauth2-proxy.cfg

```diff
@@ -0,0 +1,14 @@
+## OAuth2 Proxy Config File
+## https://github.com/oauth2-proxy/oauth2-proxy
+
+email_domains = [
+    "*"
+]
+
+skip_jwt_bearer_tokens = true
+
+cookie_secret = "Dhf3pGspLQ5DjtzIz_la8mq2MkCsXzeV-js-HpeRXtA="
```

Comment:
Should this be set to some value that makes it clear this is a public dummy credential?